mevolib/fetch/BioSeqs.py

```diff
@@ -52,82 +28,60 @@
 from Bio import SeqIO, Entrez, Alphabet
 
 from mevolib._utils import get_abspath
-from mevolib._py3k import StringIO, viewkeys, viewvalues, viewitems
 
 
-# Dictionary with the corresponding retrieval type of each Entrez
-# database supported by BioSeqs class
-ENTREZ_DB_DICT = {'nuccore': 'gbwithparts', 'nucest': 'gb', 'nucgss': 'gb',
-                  'popset': 'gb', 'protein': 'gp'}
-# Constant that limits the maximum number of sequences that can be
-# fetched
+# Dictionary with the corresponding retrieval type of each Entrez database supported by BioSeqs class
+ENTREZ_DB_DICT = {'nuccore': 'gbwithparts', 'nucest': 'gb', 'nucgss': 'gb', 'popset': 'gb', 'protein': 'gp'}
+# Constant that limits the maximum number of sequences that can be fetched
 MAX_NUM_SEQS = 100000
 
 
-def _get_entrez_db_rettype (entrez_db):
-    """
-    Check if the given entrez database is supported by the BioSeqs class.
-    Supported entrez databases are stored in 'ENTREZ_DB_DICT'.
-
-    Arguments:
-        entrez_db  (string)
-            Entrez database name.
+def _get_entrez_db_rettype (entrez_db: str) -> str:
+    """Checks if the given entrez database is supported by the BioSeqs class. Supported entrez databases 
+    are stored in 'ENTREZ_DB_DICT'.
+    Returns corresponding retrieval type for the given entrez database.
```

Comment:
```suggestion
    """Checks if the given entrez database is supported by the BioSeqs class.
    
    Supported Entrez databases are stored in `ENTREZ_DB_DICT`.
```
As before, the "Returns" bit should go to its own section.